src/scalarstats.jl

```diff
@@ -257,6 +334,14 @@ zscore!{T<:AbstractFloat}(X::AbstractArray{T}, μ::Real, σ::Real) = _zscore!(X,
 zscore!{T<:AbstractFloat,U<:Real,S<:Real}(X::AbstractArray{T}, μ::AbstractArray{U}, σ::AbstractArray{S}) =
     (_zscore_chksize(X, μ, σ); _zscore!(X, X, μ, σ))
 
+
+"""
+    zscore(X, [μ,] [σ]) -> Array
+
+Compute the z-scores of `X`, optionally specifying a precomputed mean `μ` and
+standard deviation `σ`. The mean and standard deviation can be real numbers or
+arrays of real numbers.
```

Comment:
Explain what happens when they are arrays. Also mention this possibility for `zscores!`